src/client/powerpages/utils/CommonUtils.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+export const isNullOrEmpty = (str: string | undefined): boolean => {
```

Comment:
There is a significant difference in null and undefined in JS. 
Keep a method for the check null and undefined.
keep a method for empty length string.

Incase you are interested to understand the difference https://www.geeksforgeeks.org/undefined-vs-null-in-javascript/

Reply:
right, but here '!str' checks for all kind of **falsy** values : false, 0, "", null, undefined, NaN. Let me know if I am missing something.